Host/Controllers/ItemController.cs

```diff
@@ -27,5 +27,12 @@ public GetItemsResponse GetItemByItemId(int id)
             var responseFromService = ItemService.GetItemById(id);
             return responseFromService;
         }
+
+        [HttpPost]
+        public string GetStudentString([FromBody] Student student)
+        {
```

Comment:
remove this

Reply:
maam i didnt get why i have to remove this 
Isnt it essential